src/store/src/Command/DropStoreCommand.php

```diff
@@ -0,0 +1,96 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\AI\Store\Command;
+
+use Psr\Container\ContainerInterface;
+use Symfony\AI\Store\Exception\RuntimeException;
+use Symfony\AI\Store\ManagedStoreInterface;
+use Symfony\Component\Console\Attribute\AsCommand;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Completion\CompletionInput;
+use Symfony\Component\Console\Completion\CompletionSuggestions;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Style\SymfonyStyle;
+
+/**
+ * @author Guillaume Loulier <personal@guillaumeloulier.fr>
+ */
+#[AsCommand(name: 'ai:store:drop', description: 'Drop the required infrastructure for the store')]
+final class DropStoreCommand extends Command
+{
+    /**
+     * @param string[] $storeNames
+     */
+    public function __construct(
+        private readonly ContainerInterface $storeLocator,
+        private readonly array $storeNames = [],
+    ) {
+        parent::__construct();
+    }
+
+    public function complete(CompletionInput $input, CompletionSuggestions $suggestions): void
+    {
+        if ($input->mustSuggestArgumentValuesFor('store')) {
+            $suggestions->suggestValues($this->storeNames);
+        }
+    }
+
+    protected function configure(): void
+    {
+        $this
+            ->addArgument('store', InputArgument::OPTIONAL, 'Name of the store to drop')
+            ->setHelp(<<<EOF
```

Comment:
AFAIK we can now add the help to the `#[AsCommand]` attribute

Reply:
Depending on the version, the `help` attribute is not always supported, should we stick to the method or just force the Sf version?

Reply:
Ah ok then let's keep it 👍🏻😊